.gitignore

```diff
@@ -1,3 +1,5 @@
 node_modules
 npm-debug.log
 .idea/
+cov-unit/
+coverage/
```

Comment:
no need to ignore coverage folder if you don't run istanbul report (see other comments)